testsuite/gnat2goto/tests/discriminated_record/test.out

```diff
@@ -0,0 +1 @@
+VERIFICATION SUCCESSFUL
```

Comment:
Note: In this test we _mostly_ model discriminated records correctly, hence we can produce a .json_symtab without having cbmc crash on reading it. We don't really do _assignments_ on variant records correctly at the moment though.